code/messaging/src/main/java/com/adobe/marketing/mobile/aepcomposeui/components/SmallImageCard.kt

```diff
@@ -51,15 +46,22 @@ fun SmallImageCard(
         }
     }
 
-    Card(
-        modifier = Modifier
-            .clickable {
-                observer?.onEvent(UIEvent.Interact(ui, UIAction.CLICK))
-            }
+    AepCardComposable(
+        defaultCardStyle = style.defaultCardStyle,
+        overriddenCardStyle = style.smallImageCardStyle,
+        onClick = {
+            observer?.onEvent(UIEvent.Interact(ui, UIAction.CLICK))
+        }
```

Comment:
Discussed offline with @spoorthipujariadobe  on 
- making the internal composables rely on a single style - instead of default and overridden style values. This can be achieved if we have the ability to merge at the SmallImageUIStyle level.  SmallImageUIStyle and other high level template styles can use a builder to accept the overridden values and then merge default values  when returning the style itself.
- possibility of dropping *Composable suffix for views.